### ISSUE TITLE
Code for the paper?

### ISSUE THREAD

**Issue by sunfanyunn** [econser/py_irsg#1]
Is this the code for the paper 
https://hci.stanford.edu/publications/2015/scenegraphs/JohnsonCVPR2015.pdf ?

**Comment by econser** [comment]
Yes, the original code was written in MATLAB.  This is a version that I put together in Python 2.7
